src/main/resources/application.properties

```diff
@@ -1 +1,20 @@
 spring.application.name=music-management-system
+server.port=8085
+
+#DB
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5433/music-db
+spring.datasource.username=postgres
+spring.datasource.password=secret
```

Comment:
Proponuję dodać (gdzieś w początkach) property `spring.profiles.active=local`, co spowoduje aktywowanie profilu `local`.

I te trzy właściwości związane z datasource (url, username, password) przenieść do pliku `application-local.properties`

Tym sposobem ogólna konfiguracja aplikacyjna nie będzie w sobie zawierała właściwości potrzebnych tylko do lokalnego uruchomienia.

To co tu napisałem wystarczy do osiągnięcia zamierzonego efektu, ale jeśli jesteś zainteresowana, więcej o springowych profilach możesz poczytać np. tu: https://www.baeldung.com/spring-profiles